docs/gaia-tutorials/join-mainnet.md

```diff
@@ -7,29 +7,41 @@ order: 3
 
 ## Quickstart
 
-**Instant Gratification Snippet**
+**Get Ready**
+Note: You'll need Go v1.16 or higher.
```

Comment:
Let's stick to 1.15 until 1.16 is proven out for our upstream/downstream builds.

Reply:
ok